src/data/python-quiz.ts

```diff
@@ -919,6 +919,15 @@ const pythonQuiz = [
       "In Python, we start a comment with a hash symbol #. A line that starts with this symbol will be interpreted as a comment in a Python program. ",
     Link: "https://forum.freecodecamp.org/t/how-to-comment-your-code-in-python-explained-with-examples/19220",
   },
+  { 
+    Question: "Which type of Programming does Python support?",
+    Answer: "# all of the mentioned", 
+    Distractor1: "structured programming", 
+    Distractor2: "object-oriented programming", 
+    Distractor3: "all of the mentioned", 
```

Comment:
```suggestion
    Distractor3: "functional", 
```

Reply:
hi @jdwilkin4 if I add "functional, " what should I add to the answer? Since Python supports structured and object-oriented too.

Reply:
I left a comment in my original review that you should use all mentioned paradigms.